.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,27 @@
-#### What is the purpose of this pull request?
-<!--- Describe your changes in detail. -->
-
 #### What problem is this solving?
+
 <!--- What is the motivation and context for this change? -->
 
-#### How should this be manually tested?
+#### How to test it?
+
+<!--- Don't forget to add a link to a Workspace where this branch is linked -->
+
+[Workspace](Link goes here!)
+
+#### Screenshots or example usage:
+
+<!--- Add some images or gifs to showcase changes in behaviour or layout. Example: before and after images -->
+
+#### Describe alternatives you've considered, if any.
+
+<!--- Optional -->
+
+#### Related to / Depends on
+
+<!--- Optional -->
+
+#### How does this PR make you feel? [:link:](http://giphy.com/)
```

Comment:
What? :thonking:

Reply:
This is something we took from the PR template that used to be used by the Site Editor and now CMS team and we found it might be a nice addition to the PRs opened by the community in Store Framework repositories. The idea is that you would add a funny GIF in this section. 

These two are examples: 

- https://github.com/vtex/admin-cms/pull/95
- https://github.com/vtex/admin-cms/pull/87

I know this repo is not maintained by us, but it was listed in the script since it's used in stores, that's why this PR was opened.

Reply:
Not too fond of this either.